roboflow/config.py

```diff
@@ -1,24 +1,72 @@
 import os
+import json
 
-CLASSIFICATION_MODEL = os.getenv("CLASSIFICATION_MODEL", "ClassificationModel")
+
+conf_location = os.getenv(
+    "ROBOFLOW_CONFIG_DIR", default=os.getenv("HOME") + "/.config/roboflow/config.json"
+)
```

Comment:
you can use `pathlib` that is compatible across different platforms (linux, osx, windows)